docs/source/developers/release.rst

```diff
@@ -216,123 +216,365 @@ Be sure to go through on the following checklist:
+   :class-container: sd-shadow-md
 
-    Select "Edit"
+   A PMC member must commit the source release artifacts to Subversion:
 
-    Set "Start date"
+   .. code-block:: Bash
 
-    Click "Save" button
+      # dev/release/post-01-upload.sh 0.1.0 0
+      dev/release/post-01-upload.sh <version> <rc>
 
-.. dropdown:: Updating the Arrow website
+.. dropdown:: Upload binary release artifacts to Artifactory
    :animate: fade-in-slide-down
    :class-title: sd-fs-5
    :class-container: sd-shadow-md
 
-    Fork the `arrow-site repository <https://github.com/apache/arrow-site>`_ and clone it next to the arrow repository.
+   A PMC member must upload the binary release artifacts to Artifactory:
 
-    Generate the release note:
+   .. code-block:: Bash
 
-    .. code-block::
-    
-        # dev/release/post-03-website 0.13.0 0.14.0
-        dev/release/post-03-website <previous-version> <version>
-    
-    Create a pull-request and a Jira with the links the script shows at the end.
+      # dev/release/post-02-binary.sh 0.1.0 0
+      dev/release/post-02-binary.sh <version> <rc number>
 
-.. dropdown:: Uploading source release artifacts to SVN
+.. dropdown:: Update website
    :animate: fade-in-slide-down
    :class-title: sd-fs-5
    :class-container: sd-shadow-md
 
-    A PMC member must commit the source release artifacts to SVN:
+   Add a release note for the new version to our website and update the latest release information:
 
-    .. code-block::
-    
-        # dev/release/post-01-upload.sh 0.1.0 0
-        dev/release/post-01-upload.sh <version> <rc>
+   .. code-block:: Bash
+
+      ## Prepare your fork of https://github.com/apache/arrow-site .
+      ## You need to do this only once.
+      # git clone git@github.com:kou/arrow-site.git ../
+      git clone git@github.com:<YOUR_GITHUB_ID>/arrow-site.git ../
+      cd ../arrow-site
+      ## Add git@github.com:apache/arrow-site.git as "apache" remote.
+      git remote add apache git@github.com:apache/arrow-site.git
+      cd -
+
+      ## Generate a release note for the new version, update the
+      ## latest release information automatically.
+      # dev/release/post-03-website.sh 9.0.0 10.0.0
+      dev/release/post-03-website.sh OLD_X.OLD_Y.OLD_Z X.Y.Z
+
+   This script pushes a ``release-note-X.Y.Z`` branch to your ``apache/arrow-site`` fork. You need to open a pull request from the ``release-note-X.Y.Z`` branch on your Web browser.
 
-.. dropdown:: Uploading binary release artifacts to Artifactory
+.. dropdown:: Update Homebrew packages
    :animate: fade-in-slide-down
    :class-title: sd-fs-5
    :class-container: sd-shadow-md
 
-    A PMC member must upload the binary release artifacts to Artifactory:
+   Open a pull request to Homebrew:
 
-    .. code-block::
-    
-        # dev/release/post-02-binary.sh 0.1.0 0
-        dev/release/post-02-binary.sh <version> <rc number>
+   .. code-block:: Bash
+
+      ## You need to run this on macOS that Homebrew is installed.
```

Comment:
Minor nit, this should also work with brewlinux, that's how I did the 9.0.0 one

Reply:
Really!? I didn't know that!

I've added "or Linux".